apps/web/components/CommunityCard.tsx

```diff
@@ -43,9 +43,8 @@ export function CommunityCard({
         width={100}
       />
       <div className="flex flex-col gap-2">
-        {/* fixed height for 2row title */}
-        <div className="flex h-[37px] items-center">
-          <h5>{name}</h5>
+        <div className="flex items-start w-fit max-w-full tooltip" data-tip={name}>
```

Comment:
I would be in favor of only having the tooltip when overflowing, I crafted you a little helper component for that with gemini if you also think this:
```
import React, { useRef } from "react";

interface TooltipOnOverflowProps {
  text: string; // The full text to display in the tooltip
  children: React.ReactNode; // The text element (with ellipsis) to trigger the tooltip
  className?: string; // Additional Tailwind classes for customization
}

export const TooltipOnOverflow: React.FC<TooltipOnOverflowProps> = ({
  text,
  children,
  className,
}) => {
  const textRef = useRef<HTMLSpanElement>(null);
  const isOverflowing =
    textRef.current &&
    textRef.current.offsetWidth < textRef.current.scrollWidth;

  return (
    <div
      className={`tooltip relative inline-block ${className}`}
      data-tip={text}
      data-tip-show={isOverflowing}
    >
      <span ref={textRef} className="truncate">
        {children}
      </span>
    </div>
  );
};

```

Reply:
usable by wraping the children instead of the div with the data-tip:
```
<TooltipOnOverflow text="Title">
  Title
</TooltipOnOverflow>
```

Reply:
Didnt tested

Reply:
Dont pass too much times if doesnt work

Reply:
gonna try it out!

Reply:
Try, not worked as expecetd, but nevertheless I find having tooltip for every community UX friendly and consistent